pnpm-lock.yaml

```diff
@@ -1,1319 +1,2644 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
```

Comment:
This is downgrading the lock file version. You may need to update pnpm.

```
$ pnpm --version
9.0.5
```

Reply:
https://github.com/Snowfork/snowbridge-app/pull/13/commits/5d26656891648fbb9e350e3663063f8cccc9ec55